### ISSUE TITLE
conflict with scout v8

### ISSUE THREAD

**Issue by maoxuner** [michaelbaril/sqlout#2]
```
Problem 1
    - Installation request for baril/sqlout ^1.1 -> satisfiable by baril/sqlout[v1.1.0].
    - baril/sqlout v1.1.0 requires laravel/scout ^7.1 -> satisfiable by laravel/scout[v7.1.0, v7.1.1, v7.1.2, v7.1.3, v7.2.0, v7.2.1] but these conflict with your requirements or minimum-stability.
```

**Comment by michaelbaril** [comment]
Hi @maoxuner , I've just released v2.0.0, which is compatible with Scout v8.
Let me know if you have any issue with it.
Thanks

**Comment by maoxuner** [comment]
> Hi @maoxuner , I've just released v2.0.0, which is compatible with Scout v8.
> Let me know if you have any issue with it.
> Thanks

I've tried the package these two days. Everything works well.
Thank you.
